### ISSUE TITLE
Values of promises and active bindings need to be accessible in debugger

### ISSUE THREAD

**Issue by int19h** [Microsoft/RTVS#749]
Right now, if a variable is a promise or an active binding, we show it as such in Locals. For promise, this means that you see the expression that it was constructed from as value, and for binding, you just see `<active binding>`.

This is good because we don't inadvertently modify program state, but we still need to let user explicitly force the promise / execute the binding getter, and display the resulting value. Without the means to do so, debugging experience can be rather vexing. For example, consider this simplifed scenario:

![image](https://cloud.githubusercontent.com/assets/1175142/12663823/00fcdeb4-c5e0-11e5-98a4-3d1283335871.png)

Here's what Locals look like:

![image](https://cloud.githubusercontent.com/assets/1175142/12663837/16166b26-c5e0-11e5-852c-8c0e111a27ee.png)

Basically, you need to manually go up the stack to figure out what `x` was in the calling frame, and for deeper nested calls you'd need to do so repeatedly until you get to a non-promise

There are significant UI limitations around this for Locals, since we can't just use the Refresh icon that appears for error values (well, we can, but then we'd have to make promises error values, and they'd get the error icon...). I think we can do that via the magnifying icon dropdown, however.
